src/Constant.js

```diff
@@ -0,0 +1,221 @@
+    IndicatorZigZagIcon,
+    IndicatorBollingerIcon,
+    IndicatorDonchianIcon,
+    IndicatorAveragesIcon,
+    IndicatorEnvelopeIcon,
+    IndicatorAlligatorIcon,
+    IndicatorFractalChaosIcon,
+    // Chart Types
+    TypeAreaIcon,
+    TypeCandleIcon,
+    TypeHollowIcon,
+    TypeOhlcIcon,
+} from './components/Icons.jsx';
+
+export const drawTools = {
+    channel:    { id: 'channel',     text: t.translate('Channel'), icon: DrawToolsChannelIcon },
+    segment:    { id: 'continuous',  text: t.translate('Continuous'), icon: DrawToolsContinuousIcon },
+    fibfan:     { id: 'fibfan',      text: t.translate('Fib Fan'), icon: DrawToolsFibonaccifanIcon },
+    horizontal: { id: 'horizontal',  text: t.translate('Horizontal'), icon: DrawToolsHorizontalIcon },
+    line:       { id: 'line',        text: t.translate('Line'), icon: DrawToolsLineIcon },
+    ray:        { id: 'ray',         text: t.translate('Ray'), icon: DrawToolsRayIcon },
+    rectangle:  { id: 'rectangle',   text: t.translate('Rectangle'), icon: DrawToolsRectangleIcon },
+    tirone:     { id: 'tirone',      text: t.translate('Trend'), icon: DrawToolsTrendIcon },
+    vertical:   { id: 'vertical',    text: t.translate('Vertical'), icon: DrawToolsVerticalIcon },
+};
+
+export const IndicatorsTree = [
+    {
+        id: 'momentum',
+        name: t.translate('Momentum'),
+        icon: IndicatorCatMomentumIcon,
+        items: [
+            { id: 'Awesome', name: t.translate('Awesome Oscillator'), description: t.translate('There isn\'t any description here.'), icon: IndicatorAwesomeOscillatorIcon },
+            { id: 'Detrended', name: t.translate('Detrended Price Oscillator'), description: t.translate('There isn\'t any description here.'), icon: IndicatorDTrendedIcon },
+            { id: 'Gator', name: t.translate('Gator Oscillator'), description: t.translate('There isn\'t any description here.'), icon: IndicatorGatorIcon },
+            { id: 'macd', name: t.translate('MACD'), description: t.translate('There isn\'t any description here.'), icon: IndicatorMacdIcon },
+            { id: 'Price ROC', name: t.translate('Rate of Change'), description: t.translate('There isn\'t any description here.'), icon: IndicatorRateChangeIcon },
+            { id: 'rsi', name: t.translate('Relative Strength Index (RSI)'), description: t.translate('There isn\'t any description here.'), icon: IndicatorRSIIcon },
+            { id: 'stochastics', name: t.translate('Stochastic Oscillator'), description: t.translate('There isn\'t any description here.'), icon: IndicatorStochasticOscillatorIcon },
+            { id: 'Stch Mtm', name: t.translate('Stochastic Momentum Index'), description: t.translate('There isn\'t any description here.'), icon: IndicatorStochasticMomentumIcon },
+            { id: 'Williams %R', name: t.translate('William\'s Percent Range'), description: t.translate('There isn\'t any description here.'), icon: IndicatorWilliamPercentIcon },
+        ],
```

Comment:
vertical alignment for theses too

Reply:
Sure @mahdipourziaei-binary I'll handle these in another card later as these all, are an improvement.